js/scorm/cookieLMS.js

```diff
@@ -89,30 +133,42 @@ export function start () {
 
     LMSInitialize: function() {
       configure();
-      if (!this.fetch()) {
-        this.data['cmi.core.lesson_status'] = 'not attempted';
-        this.data['cmi.suspend_data'] = '';
-        this.data['cmi.core.student_name'] = 'Surname, Sam';
-        this.data['cmi.core.student_id'] = 'sam.surname@example.org';
-        this.store(true);
-      }
+      this.initialize({
+        'cmi.interactions': [],
+        'cmi.core.lesson_status': 'not attempted',
+        'cmi.suspend_data': '',
+        'cmi.core.student_name': 'Surname, Sam',
+        'cmi.core.student_id': 'sam.surname@example.org'
+      });
       return 'true';
     },
 
     LMSFinish: function() {
       return 'true';
     },
 
-    LMSGetValue: function(key) {
-      return this.data[key];
+    LMSGetValue: function(path) {
```

Comment:
Can't we just call `this.GetValue(path, value)` if the logic is identical
Likewise for LMSSetValue and this.SetValue()

Reply:
Yes. Agreed.